src/pages/bookReview.tsx

```diff
@@ -0,0 +1,62 @@
+import { useLocation } from "react-router-dom";
+import styled from "styled-components";
+import { StarRating } from "../component/starRating";
+import { useGetBookById } from "../hooks/getBookbyId";
+
+const Container = styled.div`
+  margin-left: 6%;
+  margin-right: 6%;
+`;
+function BookReview() {
+  let location = useLocation();
+  const id = location.state.id;
+
+  const bookInfo = useGetBookById(id);
+
+  return (
+    <Container>
+      {bookInfo && (
+        <div>
+          <div>
+            <img src={bookInfo?.volumeInfo.imageLinks.thumbnail} alt="" />
```

Comment:
this `?.` is unneeded because `bookInfo &&` will make sure that `bookInfo` is already defined (it will not be null or undefined) it will be truthy